app/models/adviser.rb

```diff
@@ -12,15 +12,18 @@ class Adviser < ActiveRecord::Base
 
   validates_acceptance_of :confirmed_disclaimer, accept: true
 
+  validates :covers_whole_of_uk,
+    inclusion: { in: [true, false] }
+
   validates :travel_distance,
     presence: true,
     inclusion: { in: TravelDistance.all },
-    unless: :covers_whole_of_uk?
+    unless: -> { covers_whole_of_uk or covers_whole_of_uk.nil? }
```

Comment:
Solid. Just a couple minor things. Can you replace `or` with `||` and I reckon it might be worth moving this logic into a predicate method, then you can use it in both places and revert to `unless: :bleh_something?` 

WDYT?